docs/data/material/customization/theme-components/theme-components.md

```diff
@@ -77,17 +90,62 @@ const finalTheme = createTheme({
 
 {{"demo": "GlobalThemeOverrideCallback.js"}}
 
-### Using `sx` (experimental) syntax
+### Using the `sx` experimental syntax
+
+If you are not familiar with the `sx` prop, make sure you first check out [the concept around it](/system/the-sx-prop/) and [the difference with the `styled` utility](/system/styled/#difference-with-the-sx-prop).
+
+To have a similar shorthand CSS notation to the `sx` prop but within the theme, you can use `sx` inside the `stylesOverrides`.
+
+{{"demo": "GlobalThemeOverrideSx.js", "defaultCodeOpen": false}}
+
+```tsx
+const finalTheme = createTheme({
+  components: {
+    MuiChip: {
+      styleOverrides: {
+        root: sx({
+          px: 1,
+          py: 0.25,
+          borderRadius: 1,
+        }),
+        label: {
+          padding: 'initial',
+        },
+        icon: sx({
+          mr: 0.5,
+          ml: '-2px',
+        }),
+      },
+    },
+  },
+});
+```
+
+### Specificity
 
-If you are not familiar `sx`, first check out [the concept](/system/the-sx-prop/) and [the difference with the `styled`](/system/styled/#difference-with-the-sx-prop).
+If you use the theming approach to customize the components, you'll still be able to override them using the `sx` prop as it has a higher CSS specificity.
```

Comment:
This isn't entirely clear. Does this mean that `sx` in the theme overrides `sx` on individual components? Or vice versa?

Reply:
Oh, this is a great question... @mnajdova should know all the System answers 😃

Reply:
`sx` > `theme variants` > `theme styleOverrides`.